packages/cms-lib/lib/config.js

```diff
@@ -142,7 +136,7 @@ const configFilenameIsIgnoredByGitignore = ignoreFiles => {
     const gitignoreContents = fs.readFileSync(gitignore).toString();
     const gitignoreConfig = ignore().add(gitignoreContents);
 
-    if (gitignoreConfig.ignores(getConfigFilename())) {
+    if (gitignoreConfig.ignores(path.basename(_configPath))) {
```

Comment:
While this works in most situations, this doesn't work in cases where the `.gitignore` rule only applies based on the path.

Reply:
This can be addressed with a followup.